### ISSUE TITLE
RGB Led in example isn't working

### ISSUE THREAD

**Issue by TomYeoman** [sarahgp/p5bots#19]

**Comment by naimenz** [comment]
Replacing 'rgba' with 'levels' fixes this. I think there must have been an update to p5 changing the color object.

**Comment by sarahghp** [comment]
Yes, I think so, too. I've updated master to use the accessor functions since the internal representation appears to keep changing, but I'll need to test it before I deploy a new version to NPM.
